### ISSUE TITLE
notiPayload - payloadValue not supporting objects

### ISSUE THREAD

**Issue by robertalexa** [Tom-Hirschberger/MMM-MQTTbridge#15]
Hi,

I have tried to implement this for my use case and found out that I can't compare the dictionary noti payload with the actual payload if I am looking for an object.

My dictionary:
```
var notiHook = [
  {
    notiId: "PILIGHTS_SEQUENCE",
    notiPayload: [
      {
        payloadValue: {sequence:"pink_static"}, 
        notiMqttCmd: ["PINK_STATIC"]
      },
    ],
  },
];
var notiMqttCommands = [
  {
    commandId: "PINK_STATIC",
    mqttTopic: "magicmirror/lights",
    mqttMsgPayload: 'pink_static'
  },
];
```
Looking at the code, there was no accommodation for comparing objects at all, and none the less the mqtt payload would have been converted .toString() anyway, which will output [object Object]

So I have made some changes to the code to make it work with objects too and it works well. I am unable to do a PR right now but I will leave the code here. Author please let me know if you would like to implement it or if you want me to do a PR when I get a moment.

In MQTT-Bridge.js in notiToMqtt function

```
if (JSON.stringify(this.config.notiDictionary.notiHook[i].notiPayload[j].payloadValue) === JSON.stringify(payload) || this.config.notiDictionary.notiHook[i].notiPayload[j].payloadValue == '') 
          {
            // if NOTI ID and Payload found - search for Commands (could be an array of commands)
            for (var k = 0; k < this.config.notiDictionary.notiHook[i].notiPayload[j].notiMqttCmd.length; k++) 
            {
              for (var x in this.config.notiDictionary.notiMqttCommands) 
              {
                if (this.config.notiDictionary.notiHook[i].notiPayload[j].notiMqttCmd[k] == this.config.notiDictionary.notiMqttCommands[x].commandId) 
                {
                  // if NOTI matched in the Dictionary, send respective MQTT message
                  // If payloadValue is empty in notiDictionary --> send payload of Notification to MQTT - Otherwise send payload defined in notiDictionary
                  if (this.config.notiDictionary.notiHook[i].notiPayload[j].payloadValue == '') 
                  {
                    this.publishNotiToMqtt(this.config.notiDictionary.notiMqttCommands[x].mqttTopic, JSON.stringify(payload));
                  } 
                  else 
                  {
                    this.publishNotiToMqtt(this.config.notiDictionary.notiMqttCommands[x].mqttTopic, JSON.stringify(this.config.notiDictionary.notiMqttCommands[x].mqttMsgPayload));
                  }
                  break;
                }
              }
            }
            break;
          }
```

So basically JSON.stringify everything and then compare them. This does have drawbacks, because if your dictionary object properties are not in the same order as the actual payload it won't work. But I felt there was no need to handle a proper intensive object comparison when a dictionary file in involved. Code should help the person, but the person should also know what they are doing if they are working with objects.

Hope this helps someone,
Rob
